pkg/apis/monitoring/v1/types.go

```diff
@@ -653,6 +653,8 @@ type RemoteWriteSpec struct {
 	// name is used in metrics and logging in order to differentiate queues.
 	// Only valid in Prometheus versions 2.15.0 and newer.
 	Name string `json:"name,omitempty"`
+	// Enables sending of exemplars over remote write. Note that exemplar-storage itself must be enabled using the enableFeature option for exemplars to be scraped in the first place.  Only valid in Prometheus versions 2.27.0 and newer.
+	SendExemplars bool `json:"sendExemplars,omitempty"`
```

Comment:
Can you make it *bool? This way, a nil value would stick to whatever default value is chosen by Prometheus.

```suggestion
	SendExemplars *bool `json:"sendExemplars,omitempty"`
```